src/UI/Component/Floatable/Factory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/* Copyright (c) 2017 Timon Amstutz <timon.amstutz@ilub.unibe.ch> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Floatable;
+
+use ILIAS\UI\Component\Component;
+/**
+ * Floatable factory
+ */
+interface Factory {
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       Instructional Overlays keep didactical instructions available to users
+	 *       while they carry out workflows presented by this instruction that spread
+	 *       over several screens. Instructional information and actions are made
+	 *       available at the very screen the user is working on.
```

Comment:
No, this takes instructions from assignments in the exercise. 
Users have to complete complex workflows that start once they click the "Add Portfolio" / "Edit Text" / "Add Blog" button in the assignment: 
operation unknown objects and creating content, under a schedule according to instructions accompanied by files

What happens to the help depend on the page layout revision and is currently not the fous of this ui